core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -507,16 +503,15 @@ describe('Exploration editor page component', function() {
         ' fn property function on index 3 of EDITOR_TUTORIAL_OPTIONS array',
       () => {
         var element = angular.element('div');
-        // @ts-ignore is being used in order to ignore JQuery properties that
-        // should be declared.
         spyOn(window, '$').and.returnValue(element);
         var animateSpy = spyOn(element, 'animate').and.callThrough();
-        // @ts-ignore Angular element method doesn't expect to receive
-        // 1 argument in the lints.
         spyOn(angular, 'element')
           .withArgs('#tutorialStateContent').and.returnValue({
-            // @ts-ignore Angular element should have more properties than
-            // just offset in the lint settings.
+            // There is a typescript error here because angular element expects
+            // around 170 more properties than just one (offset). We are
+            // suppressing this error because we need only offset for testing
+            // purposes.
+            // @ts-ignore
```

Comment:
ts-expect-error? (here + below) 

I think that's the general thing to use for these cases.

Reply:
done